api/core/utils/schema_sanitation.py

```diff
@@ -113,7 +113,7 @@ def model_schema(self, *args: Any, **kwargs: Any):
         return generated
 
 
-def _add_missing_defs(schema: dict[str, Any]) -> dict[str, Any]:
+def add_missing_defs(schema: dict[str, Any]) -> dict[str, Any]:
```

Comment:
just use `streamline_schema` ?

Reply:
This code is deprecated and should be removed soon

Reply:
OK I've added a deprecated decorator and migrated to streamline_schema.